README.rst

```diff
@@ -186,7 +186,7 @@ There are multiple supported backends, which can be selected at runtime in the e
 +----------------------------+---------------------------------------------------+-----------------------+
 | ``/gpu/cuda/gen``          | Optimized pure CUDA kernels using code generation | No                    |
 +----------------------------+---------------------------------------------------+-----------------------+
-| HIP Native Backend                                                                                     |
+| HIP Native Backends                                                                                    |
```

Comment:
Some projects have a policy to always use plural for headings that may become lists. It reads fine when there's only one and doesn't require someone to notice it later.